### PR TITLE
Group chat feature

### PR DESCRIPTION
**Pull request by nbhamilton94**

### CODE REVIEW

File: src/pages/groupView/groupView.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			</ion-row>
+		</ion-grid>
+	</ion-toolbar>
+</ion-header>
+<!-- Main content which contains the groups. -->
+<ion-content padding>
+	<!-- Iterate through all the groups. --> 
+	<ion-list *ngFor="let group of groups" no-lines>
+		<ion-item > 
+			<p class="groupname" >{{ group.name }}</p>
```
Comment (khen27) [inline src/pages/groupView/groupView.html]: maybe add a comment here

File: src/pages/groupView/groupView.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	//defining groups, which would be an array of objects in this example.
+	//groupChatView: for the purpose of having navigation to the groupChatView page.
+	groups : Array<Object>;
+	groupChatView = GroupChatView;
+	value = 1;
+	constructor(public navCtrl: NavController) {
+		//Sample groups are being created in the constructor to be rendered in groupView.html.
+		this.initializeGroups();
+	}
+	/*
```
Comment (khen27) [inline src/pages/groupView/groupView.ts]: should there be a space in between your comments and the code above?

File: src/pages/groupView/groupView.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+		of filtered groups.
+  	*/
+  	getGroups(ev: any){
+  		//Reset items back to all of the items.
+    	this.initializeGroups();
+
+  		//Set val to the value of the searchbar.
+  		let val = ev.target.value;
+
+  		if (val && val.trim() != '') {
```
Comment (khen27) [inline src/pages/groupView/groupView.ts]: this code looks dangerous - I would clean this one up, not sure what it's supposed to be....

if((val && val.trim()) != ' ') => I'm assuming you don't mean this logic

if((val != ' ') && (val.trim() != ' ')) => I'm assuming you do mean this logic

**Comment by grant-braqs** [comment]
Always type a short description for the pull request. Here you can highlight focus points.

File: src/pages/groupView/groupView.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  			this.groups = this.groups.filter((group) => {
+  				return this.filterByGroup(group,val);
+  			});
+    	}
+  	}
+
```
Comment (grant-braqs) [inline src/pages/groupView/groupView.ts]: Comment on this method. What needs to happen to add a group.

**Comment by grant-braqs** [comment]
@nbhamilton94 This is the "GroupChatFeature," but the code seems to be geared towards a GroupView.

If you can, rename the branch and pull again. If you need help, let me know.

**Comment by grant-braqs** [comment]
Closing pull request. The Pull Request here is dealing with the master, so I am closing now. The correct pull request has been made. 

-Grant

**Pull request closed without merging**